nanoFramework.Device.Bluetooth/IO/Buffer.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Text;
+
+namespace nanoFramework.Device.Bluetooth
+{
+    /// <summary>
+    /// Represents a referenced array of bytes used by byte stream read and write interfaces.
+    /// </summary>
+    public class Buffer
```

Comment:
why not use SpanByte instead?

Reply:
I was trying to keep as near to original.
I have a look at this. Maybe we can discuss this in later comment

Reply:
I've look at it again and the way you are using it is ok like this. I guess it's more for read and write where rather than a byte[] having a SpanByte would make sense.